src/views/Login.vue

```diff
@@ -50,21 +51,32 @@ export default {
     return {
       username: '',
       password: '',
+      error: ''
     };
   },
   methods: {
     login() {
+    this.error = '';
       const payload = {
         username: this.username,
         password: this.password,
       };
       this.$store
         .dispatch('login', payload)
-        //makeLogin(payload)
         .then(() => {
           this.$router.push('/');
         })
-        .catch(error => console.error(error));
+        .catch(error => {
+          const ERROR_MESSAGES = {
+            'auth-error': 'User or password incorrect',
```

Comment:
I have put the message here, but I think it would be correct for the server to return it in the future instead of having it here.

Reply:
I won't forget to do it in a future, thanks.